testsuite/forge/src/backend/local/node.rs

```diff
@@ -125,12 +125,25 @@ impl LocalNode {
             )
         })?;
 
-        println!(
+        // We print out the commands and PIDs for debugging of local swarms
+        info!(
             "Started node {:?} (PID: {:?}) with command: {:?}",
             self.name,
             process.id(),
             node_command
         );
+
+        // We print out the API endpoints of each node for local debugging
+        info!(
+            "Node {:?} REST API is listening at: 127.0.0.1:{:?}/v1",
+            self.name,
+            self.config.api.address.port()
+        );
+        info!(
+            "Node {:?} Inspection service is listening at 127.0.0.1:{:?}",
```

Comment:
The ports should be able to be printed with display right?  Same with the name.

Just trying to make sure it doesn't end up printing out weird

Reply:
Yeah, they're u64 so it doesn't matter much. Will update.